src/test/java/model/OrderTest.java

```diff
@@ -0,0 +1,38 @@
+package model;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class OrderTest {
+    @Test
+    void getProduct_ShouldReturn_OrderedProduct_ForTure(){
```

Comment:
```suggestion
    void getProduct_ShouldReturn_OrderedProduct_ForTrue(){
```

Super Testname!

Reply:
Auch bei den anderen Namen - gefällt mir gut weil

- Methodenname enthalten
- Should Logik
- Beschreibung was passieren sollte mit einem true/ false
- Auch: Das Schema ist konstant durchgezogen bei allen Methoden :heavy_check_mark: